ci/check_style.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+rapids-logger "Create checks conda environment"
+# TODO: Move this
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --generate conda \
+  --file_key checks \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" > env.yaml
+
+mamba env create -f env.yaml -n checks
```

Comment:
```suggestion
rapids-mamba-retry env create -f env.yaml -n checks
```

This step seems to fail sometimes without `rapids-mamba-retry`. I added a retry for `ConnectionError`s [here](https://github.com/rapidsai/gha-tools/commit/eb4a92c087e3f5789628b6636048b75b8dce8311)

- https://github.com/rapidsai/rmm/actions/runs/3136315901/jobs/5093520359